pkg/scheduler/framework/session_plugins.go

```diff
@@ -626,6 +626,32 @@ func (ssn *Session) TaskOrderFn(l, r interface{}) bool {
 	return helpers.CompareTask(lv, rv)
 }
 
+// NodesWherePreemptionMightHelp filter out those node that has unscheduleandunre
+func (ssn *Session) NodesWherePreemptionMightHelp(task *api.TaskInfo) []*api.NodeInfo {
```

Comment:
Suggest to change function name to `getUnschedulableAndUnresolvableNodesForTask`

Reply:
Done.